spring-cloud-starter-alibaba/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/endpoint/NacosConfigEndpointAutoConfiguration.java

```diff
@@ -52,7 +53,7 @@ public NacosConfigEndpoint nacosConfigEndpoint() {
 
 	@Bean
 	@ConditionalOnMissingBean
-	@ConditionalOnProperty(name = "spring.cloud.nacos.config.health.enabled", matchIfMissing = true)
```

Comment:
Also need to delete it at [additional-spring-configuration-metadata.json](https://github.com/alibaba/spring-cloud-alibaba/blob/master/spring-cloud-starter-alibaba/spring-cloud-starter-alibaba-nacos-config/src/main/resources/META-INF/additional-spring-configuration-metadata.json#L59).

Reply:
fixed